### DISCUSSION TITLE
paddle学习第一步就卡住，试了很多机器都是一样的问题，会不会是官方安装方法有问题，求帮助看一下，谢谢

### DISCUSSION THREAD

**Discussion by woodgreat** [PaddlePaddle/PaddleOCR#12658] [Q&A]
官方文档中写到：

1.1 安装依赖库
In [5]
# 安装依赖库
!pip install python-docx
!pip install pypinyin
!pip install LAC
!pip install --upgrade paddlenlp
!pip install pymupdf
!pip install --upgrade paddleocr
# 首次更新完以后，重启后方能生效

 

这里安装包时，pip install --upgrade paddleocr ，一直出错，上面几步都没有问题，

 

File "C:\Python311\Lib\tarfile.py", line 2100, in extract
self._extract_member(tarinfo, os.path.join(path, tarinfo.name),
File "C:\Python311\Lib\tarfile.py", line 2173, in _extract_member
self.makefile(tarinfo, targetpath)
File "C:\Python311\Lib\tarfile.py", line 2214, in makefile
with bltn_open(targetpath, "wb") as target:
^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '.\\mupdf-1.20.3-source\\thirdparty\\harfbuzz\\test\\shaping\\

请问要怎么解决

这个 pip install --upgrade paddleocr 为什么不是单纯的安装包，还要编译别的东西，有点奇怪啊

**Comment by woodgreat** [comment]
[pip_install_error.txt](https://github.com/PaddlePaddle/PaddleOCR/files/13040240/pip_install_error.txt)

这是最新的出错内容

**Comment by Gmgge** [comment]
可以在issue搜索下mupdf，有相应的解决方案可以尝试

**Comment by woodgreat** [comment]
MUFDF没有问题，已经安装成功了，就是“pip install --upgrade paddleocr”
这一步不行
